quickevent/app/plugins/Event/src/eventplugin.cpp

```diff
@@ -869,6 +871,7 @@ bool EventPlugin::closeEvent()
 	setEventName(QString());
 	QF_SAFE_DELETE(m_eventConfig)
 	//emit eventOpened(eventName()); //comment it till QE can load event with invalid name
+	emit eventClosed();
```

Comment:
`setEventOpen(false)`

Reply:
I see. Why do we have `eventOpened` event then? Should I rewrite it too?

Reply:
it is up on you, specialized signal can save some lambdas, however the lower number of signals is better :)

Reply:
redundant events removed